docs/task-setup.md

```diff
@@ -0,0 +1,23 @@
+# Installing Task on your machine
+
+## Using Homebrew:
+
+```bash
+brew install go-task/tap/go-task
+```
```

Comment:
let's move this installation instruction to `docs/local-environment.md#setup-local-environment` 
+add link to task documentation
+ delete this file